pkg/core/consensus/collector.go

```diff
@@ -72,7 +72,7 @@ func (r *roundCollector) Collect(m message.Message) error {
 
 // InitAcceptedBlockUpdate init listener to get updates about lastly accepted block in the chain
 func InitAcceptedBlockUpdate(subscriber eventbus.Subscriber) (chan block.Block, uint32) {
-	acceptedBlockChan := make(chan block.Block)
+	acceptedBlockChan := make(chan block.Block, 500)
```

Comment:
Please extrapolate the size of the channel buffer into a variable (which later, in a separate issue, will be filled at `init` time with some configuration)